doc/classes/Animation.xml

```diff
@@ -561,7 +561,7 @@
 			<param index="1" name="path" type="NodePath" />
 			<description>
 				Sets the path of a track. Paths must be valid scene-tree paths to a node and must be specified starting from the parent node of the node that will reproduce the animation. Tracks that control properties or bones must append their name after the path, separated by [code]":"[/code].
-				For example, [code]"character/skeleton:ankle"[/code] or [code]"character/mesh:transform/local"[/code].
+				For example, [code]"character/skeleton:ankle"[/code] or [code]"character/mesh:transform/local"[/code]. See also [member AnimationMixer.root_motion_track].
```

Comment:
The reference is opposite because root_motion_track depends on the format of the Track in Animation.

Reply:
I cross referenced it both directions

Reply:
Since the same document is not needed in two places, only one link is sufficient. Also unnecessary links confuse people who are not interested in root_motion but came to read the animation documentation.

Reply:
Then please cherry pick the parts that you want.